src/clientManager.js

```diff
@@ -110,7 +110,7 @@ ClientManager.prototype.removeUser = function(name) {
 ClientManager.prototype.autoload = function(/* sockets */) {
 	var self = this;
 	setInterval(function() {
-		var loaded = _.pluck(
+		var loaded = _.map(
 			self.clients,
```

Comment:
Unless I am being mistaken (it's late :-)), it seems like this is the same as:

```js
self.clients.map(function (client) {
  return client.name;
});
```

Example:

```js
> var arr = [{foo: 1}, {foo: 2}, {foo: 3}];
> _.map(arr, "foo");
[ 1, 2, 3 ]
> arr.map(function (x) { return x.foo; });
[ 1, 2, 3 ]
```

While I understand you are merely upgrading as-is here, I believe we should leverage vanilla JS whenever possible, and this is a good case :-)
What do you think?